media-video/webcamoid/webcamoid-9999.ebuild

```diff
@@ -0,0 +1,88 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake xdg-utils
+
+DESCRIPTION="A full featured webcam capture application"
+HOMEPAGE="https://webcamoid.github.io"
+if [[ ${PV} = 9999 ]]; then
+	EGIT_REPO_URI="https://github.com/webcamoid/webcamoid.git"
+	EGIT_BRANCH="master"
+	inherit git-r3
+else
+	SRC_URI="https://github.com/webcamoid/${PN}/archive/refs/tags/${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~arm64 ~x86"
+fi
+RESTRICT="mirror"
```

Comment:
why RESTRICT="mirror"
for GPL?

Reply:
I just copied 9.0.0 and it was there already, didnät think of it
It should be removed on both.

Reply:
At the time when I wrote the 9.0.0 ebuild it was on my own repo so the download process would keep failing with mirroring enabled

Reply:
Oh yea, I focused too much on the `CMAKE*` parts I didn't see this. Will remove it from 9.0.0.

Reply:
It wasn't there until today.  The pull request just got approved today. The ebuild was on my repo for almost a month(doesn't matter though).  Besides that... it wouldn't download with mirroring enabled for my personal repo because it wasn't on the GENTOO-MIRRORS at the time: https://devmanual.gentoo.org/eclass-reference/ebuild.  Search keyword RESTRICT and look at info for mirror

Reply:
> I just copied 9.0.0 and it was there already, didn't think of it
> It should be removed on both.

New here. I get it now. I looked at https://devmanual.gentoo.org/general-concepts/mirrors/#automatic-mirroring.  Didn't know it would fail to download from the mirrors the first time, then pull from upstream, then mirror it onto the GENTOO_MIRRORS so it will be there the next time. Thanks for the tip!